### PR TITLE
Add prerequisites for linear regression and MNIST tutorials.

### PR DESCRIPTION
**Pull request by indhub**

### CODE REVIEW

File: docs/tutorials/prerequisites/install-graphviz.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+## Install Graphviz
+#### On Ubuntu
+```
+sudo apt-get install graphviz
```
Comment (nswamy) [inline docs/tutorials/prerequisites/install-graphviz.md]: Can you please move this to the main Installation page? This should go for PIP/VirtualEnvironment and Linux. The Installation script for OS X handles this.

wouldn't pip install graphviz be sufficient?

Comment (piiswrong) [inline docs/tutorials/prerequisites/install-graphviz.md]: are you sure? Isn't pip install graphviz enough?

File: docs/tutorials/python/linear-regression.md [hunk 2]
```diff
@@ -11,6 +11,13 @@ import mxnet as mx
 import numpy as np
 ```
 
+## Prerequisites
+1. [MXNet](http://mxnet.io/get_started/install.html), [Graphviz](../prerequisites/install-graphviz.html).
+2. Jupyter Notebook and Graphviz python packages.
+   ```
+   pip install jupyter graphviz
```
Comment (nswamy) [inline docs/tutorials/python/linear-regression.md]: If we still need to pip install graphviz, can you move it to the main page. Visualizing the graph is a pretty common usecase and can go on the main installation page.

File: docs/tutorials/python/linear-regression.md [hunk 3]
```diff
@@ -11,6 +11,13 @@ import mxnet as mx
 import numpy as np
 ```
 
+## Prerequisites
+1. [MXNet](http://mxnet.io/get_started/install.html), [Graphviz](../prerequisites/install-graphviz.html).
+2. Jupyter Notebook and Graphviz python packages.
```
Comment (nswamy) [inline docs/tutorials/python/linear-regression.md]: Can we add a link to the Jupyter page and other packages that you sue?
[Jupyter Notebook](http://jupyter.org/)

File: docs/tutorials/prerequisites/install-graphviz.md (hunk 1 above)
Comment (indhub) [inline docs/tutorials/prerequisites/install-graphviz.md]: Yes. Tested it on Ubuntu.

Comment (indhub) [inline docs/tutorials/prerequisites/install-graphviz.md]: This is the error I get:
```
---------------------------------------------------------------------------
ExecutableNotFound                        Traceback (most recent call last)
/home/ubuntu/anaconda3/envs/mx2/lib/python2.7/site-packages/IPython/core/formatters.pyc in __call__(self, obj)
    336             method = get_real_method(obj, self.print_method)
    337             if method is not None:
--> 338                 return method()
    339             return None
    340         else:

/home/ubuntu/anaconda3/envs/mx2/lib/python2.7/site-packages/graphviz/files.pyc in _repr_svg_(self)
     99 
    100     def _repr_svg_(self):
--> 101         return self.pipe(format='svg').decode(self._encoding)
    102 
    103     def pipe(self, format=None):

/home/ubuntu/anaconda3/envs/mx2/lib/python2.7/site-packages/graphviz/files.pyc in pipe(self, format)
    118         data = text_type(self.source).encode(self._encoding)
    119 
--> 120         outs = backend.pipe(self._engine, format, data)
    121 
    122         return outs

/home/ubuntu/anaconda3/envs/mx2/lib/python2.7/site-packages/graphviz/backend.pyc in pipe(engine, format, data, quiet)
    154     except OSError as e:
    155         if e.errno == errno.ENOENT:
--> 156             raise ExecutableNotFound(args)
    157         else:  # pragma: no cover
    158             raise

ExecutableNotFound: failed to execute ['dot', '-Tsvg'], make sure the Graphviz executables are on your systems' PATH

Out[6]:
<graphviz.dot.Digraph at 0x7f62fddd07d0>
```

Comment (indhub) [inline docs/tutorials/prerequisites/install-graphviz.md]: Nope. With just the pip install, I get this error: https://github.com/dmlc/mxnet/pull/6617#discussion_r121017300

Comment (piiswrong) [inline docs/tutorials/prerequisites/install-graphviz.md]: Then move the prerequites section in each tutorial to here too.

File: docs/tutorials/vision/large_scale_classification.md [hunk 4]
```diff
@@ -244,30 +251,3 @@ It is often straightforward to achieve a reasonable validation accuracy, but ach
 If the batch size is too big, it can exhaust GPU memory. If this happens, you’ll see the error message “cudaMalloc failed: out of memory” or something similar. There are a couple of ways to fix this:
 - Reduce the batch size.
 - Set the environment variable `MXNET_BACKWARD_DO_MIRROR` to 1. It reduces the memory consumption by trading off speed. For example, with batch size 64, inception-v3 uses 10G memory and trains 30 image/sec on a single K80 GPU. When mirroring is enabled, with 10G GPU memory consumption, we can run inception-v3 using batch size of 128. The cost is that, the speed reduces to 27 images/sec.
-
-## Appendix
-### Installing OpenCV
```
Comment (piiswrong) [inline docs/tutorials/vision/large_scale_classification.md]: ?

File: docs/tutorials/prerequisites/install-graphviz.md (hunk 1 above)
Comment (indhub) [inline docs/tutorials/prerequisites/install-graphviz.md]: We are moving Graphviz installation instruction to the MXNet installation page. Once that is done, we can remove graphviz installation instructions from the tutorials.

File: docs/tutorials/python/linear-regression.md (hunk 2 above)
Comment (indhub) [inline docs/tutorials/python/linear-regression.md]: https://github.com/dmlc/mxnet/pull/6627

File: docs/tutorials/vision/large_scale_classification.md (hunk 4 above)
Comment (indhub) [inline docs/tutorials/vision/large_scale_classification.md]: Turns out `pip install opencv-python` is sufficient. So, removing this section and adding the pip install to requirements section.

**Pull request closed without merging**